docs/Installing.asciidoc

```diff
@@ -67,6 +67,7 @@ From there, you can trigger a new job or clone an existing one, e.g.:
 
 [source,sh]
 ----
+wget https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-example/main/scenario-definitions.yaml
```

Comment:
Alternatively we could encourage using the `async=1` flag. Then openQA can download the file itself on the fly.

```
openqa-cli schedule --monitor \
    async=1 SCENARIO_DEFINITIONS_YAML_FILE=https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-example/main/scenario-definitions.yaml \
    DISTRI=example VERSION=0 FLAVOR=DVD ARCH=x86_64 \
    TEST=simple_boot _GROUP_ID=0 BUILD=test \
    CASEDIR=https://github.com/os-autoinst/os-autoinst-distri-example.git \
    NEEDLES_DIR=%%CASEDIR%%/needles
```

Reply:
Are there any disadvantages to it?

Reply:
```openqa-cli schedule --help
Usage: openqa-cli schedule [OPTIONS] DISTRI=… VERSION=… FLAVOR=… ARCH=… [ISO=… …]

Options:
      --apibase <path>           API base, defaults to /api/v1
      --apikey <key>             API key
      --apisecret <secret>       API secret
      --host <host>              Target host, defaults to http://localhost
  -h, --help                     Show this summary of available options
      --osd                      Set target host to http://openqa.suse.de
      --o3                       Set target host to https://openqa.opensuse.org
      --param-file <param=file>  Load content of params from files instead of
                                 from command line arguments. Multiple params
                                 may be specified by adding the option
                                 multiple times
  -m, --monitor                  Wait until all jobs are done/cancelled and return
                                 non-zero exit code if at least on job has not
                                 passed/softfailed
      --name <name>              Name of this client, used by openQA to
                                 identify different clients via User-Agent
                                 header, defaults to "openqa-cli"
  -i, --poll-interval <seconds>  Specifies the poll interval used with --monitor
  -p, --pretty                   Pretty print JSON content
  -q, --quiet                    Do not print error messages to STDERR
  -v, --verbose                  Print HTTP response headers
```

`async=1` is not mentioned in the help and it's not immediately obvious what it does (from a user perspective). 

Does it just download the file or does it do other things as well? From the code snippet it looks like it is hard-coded to always download this specific file, i.e. if I specify a different URL it will not download the file I expect?

Reply:
The CLI is just a client. Its help doesn't contain information about the server-side parameters. This particular flag is documented in the openQA documentation if I remember correctly.

Not sure what makes you think any paths are hardcore. Of course it'll download the file you specify. 

---

> Are there any disadvantages to it?

With the suggested change the example doesn't show anymore how to specify a local path which can also be very useful to test local modifications. Maybe we still want to have such an example somewhere in the documentation.

Reply:
I missed the part where the change is in the docs, not the codebase, my bad 😅

Reply:
I still don't find any clear explanation [here](https://open.qa/docs/) of what `async=1` does in this case. It does download it on the server side, right?

Reply:
Iso.pm enqueues a minion job if async is set

Reply:
@perlpunk So the note at the bottom of https://open.qa/docs/#_spawning_multiple_jobs_based_on_templates_isos_post is not good enough? It explains what `async=1` is intended to be used for.

And yes, it does not mention the `SCENARIO_DEFINITIONS_YAML_FILE`. Maybe we should add a note where we introduce the `SCENARIO_DEFINITIONS_YAML_FILE` parameter that one can specify an HTTP-URL with the `async=1` parameter and that the download is then done server-side.

Reply:
#5855